dns/ddclient/src/opnsense/scripts/ddclient/lib/account/duckdns.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
Mind to add a license header like in the other files? Also for AWS. Noticed it was missing.

Reply:
Sorry, I'm unclear- are you asking me if you want me to add a license file, or are you suggesting you want to add one?

Reply:
just the header like:

https://github.com/opnsense/plugins/blob/caf466b7a75e8ed1241b52d3c7201c62a5f7682f/dns/ddclient/src/opnsense/scripts/ddclient/lib/account/azure.py#L1-L24

It makes the license and your contribution clearer

Reply:
Sure, I can do that. I'd prefer not to put my email address in a public repo - is there an alternative like linking to my Github page?

Reply:
just the name is also fine. GitHub handle or links are discouraged because direct communication is not possible and handle is ambiguous

Reply:
Not sure what happened to my comment...

Roger, I added a license header and I think I fixed the permissions on the 2 .py files.

Reply:
no file change no file comment no worries ;)